src/ric/d_19EE0.c

```diff
@@ -0,0 +1,154 @@
+#include "ric.h"
+
+s32 D_80155EE0[] = {0x00000F80, 0x00000100, 0x00000700, 0x00000880};
+
+s16 D_80155EF0[] = {0x0400, 0x06AB, 0x0955, 0x0C00, 0x0EAB, 0x1155};
+
+SVECTOR D_80155EFC = {-8, -72, -4, 0};
+
+SVECTOR D_80155F04 = {8, -72, -4, 0};
+
+SVECTOR D_80155F0C = {-48, -32, -4, 0};
+
+SVECTOR D_80155F14 = {-8, -32, -4, 0};
+
+SVECTOR D_80155F1C = {8, -32, -4, 0};
+
+SVECTOR D_80155F24 = {48, -32, -4, 0};
+
+SVECTOR D_80155F2C = {-48, -16, -4, 0};
+
+SVECTOR D_80155F34 = {-8, -16, -4, 0};
+
+SVECTOR D_80155F3C = {8, -16, -4, 0};
+
+SVECTOR D_80155F44 = {48, -16, -4, 0};
+
+SVECTOR D_80155F4C = {-8, 72, -4, 0};
+
+SVECTOR D_80155F54[] = {8, 72, -4, 0, -24, -48, -4, 0};
+
+SVECTOR D_80155F64 = {-8, -48, -4, 0};
+
+SVECTOR D_80155F6C[] = {8, -48, -4, 0, 24, -48, -4, 0};
+
+SVECTOR D_80155F7C = {-24, -32, -4, 0};
+
+SVECTOR D_80155F84 = {24, -32, -4, 0};
+
+SVECTOR D_80155F8C = {-24, -16, -4, 0};
+
+SVECTOR D_80155F94[] = {24, -16, -4, 0, -24, 0, -4, 0};
+
+SVECTOR D_80155FA4 = {-8, 0, -4, 0};
+
+SVECTOR D_80155FAC[] = {8, 0, -4, 0, 24, 0, -4, 0};
```

Comment:
SVECTOR is
```
typedef struct {
    short vx, vy, vz; // Vector coordinates
    short pad;        // System reserved
} SVECTOR;            // size = 0x8
```

The usage looks like
```
    vectors_ptr = &D_8015607C;
    gte_stsxy2(&prim->x0);
    gte_stszotz(&z);
    self->hitboxOffX = prim->x0 - self->posX.i.hi;
    self->hitboxOffY = prim->y0 - self->posY.i.hi;
    for (i = 0; i < 46; i++, prim = prim->next, vectors_ptr += 4) {
        gte_ldv3(vectors_ptr[0], vectors_ptr[1], vectors_ptr[3]);
        gte_rtpt();
        temp_a3 = vectors_ptr[2];
```

<img width="505" alt="image" src="https://github.com/user-attachments/assets/93e8dc8e-4eb6-46c0-97ba-5de49e817286">

So I don't think these ones are arrays, probably unused data

Reply:
Addressed